src/pages/getting-started/installing-livechat/android-widget/index.mdx

```diff
@@ -319,6 +324,59 @@ android: configChanges = "orientation|screenSize";
 
 The chat widget will handle the orientation change by itself.
 
+# Localisation
+
+You can change or localize error messages, by defining your own string resources with following id's:
+
+```java
+<string name="failed_to_load_chat">Couldn\'t load chat.</string>
+<string name="cant_share_files">File sharing is not configured for this app</string>
+<string name="reload_chat">Reload</string>
+```
+
+### Migrating to version >= 2.2.0
+
+- ChatWindowView is now interface that can be casted to View
+- `setUpWindow(configuration);` is replaced by `setConfiguration(configuration);`
+- `setUpListener(listener)` is replaced by `setEventsListener(listener)`
+- `ChatWindowView.clearSession(Context)` is moved to `ChatWindowUtils.clearSession(Context)`
+- `ChatWindowView.createAndAttachChatWindowInstance(Activity)` is moved to `ChatWindowUtils.createAndAttachChatWindowInstance(getActivity())``
+
+# SnapCall integration
+
+SnapCall integration requires AUDIO and VIDEO permissions. In order to allow your users to use SnapCall integration you need to:
+
+1. Set up your ChatWindowView Event listener, check [ChatWindowEventsListener](#ChatWindowEventsListener).
+2. Add following permissions to you app `AndroidManifest.xml` file
+   
+```xml
+    <uses-permission android:name="android.permission.RECORD_AUDIO" />
+    <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS" />
+    <uses-permission android:name="android.permission.CAMERA" />
+
+```
+
+1. Override `void onRequestAudioPermissions(String[] permissions, int requestCode)` to ask the user for permissions, as in the example below:
+   
+```java
+@Override
+public void onRequestAudioPermissions(String[] permissions, int requestCode) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+        this.requestPermissions(permissions, requestCode);
+    }
+}
+```
+
+4. Override your activity `void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults)` to pass result to `ChatWindowView`
```

Comment:
I am not sure if it is possible to write it differently, but when you look at this sentence in the preview, it is too long, and it 'enters' the area of a sidebar 🤔

Reply:
This is a bug :D Please create a Jira ticket (type bug) for this kind of issue. In this PR, try to break this line (enter + shift)